src/index.css

```diff
@@ -17,6 +17,9 @@
   --absent-cell-bg-color: theme('colors.slate.400');
   --correct-cell-bg-color: theme('colors.green.400');
   --present-cell-bg-color: theme('colors.yellow.400');
+  --wrong-char-cell-bg-color: theme('colors.purple.600');
```

Comment:
Is there a reason we are using the higher weighted color (600 over 400)?

Reply:
Coloring mainly, the different colors dont match the brightness of the yellow or green that is used in the base cases.